.gitignore

```diff
@@ -1,48 +1,6 @@
-bin/*
-!bin/ai
-bin/ai/*
-!bin/ai/compat*.nut
-!bin/ai/regression
-!bin/data
-bin/baseset/*
-!bin/baseset/openttd.grf
-!bin/baseset/opntitle.dat
-!bin/baseset/orig_extra.grf
-!bin/baseset/orig_*.obg
-!bin/baseset/orig_*.obs
-!bin/baseset/no_sound.obs
-!bin/baseset/no_music.obm
-!bin/baseset/orig_*.obm
-!bin/game
-bin/game/*
-!bin/game/compat*.nut
-!bin/scripts
-bin/scripts/*
-!bin/scripts/*.example
-!bin/scripts/readme.txt
-
-bundle/*
-bundles/*
+/.vs
```

Comment:
We've never allowed editor-specific ignores in the past, why this one?

Reply:
It was `project/.vs` in previous .gitignore, and we support Visual Studio. And my other addition is also a VS specific one.

Reply:
Well yes, but by that definition we (now) support everything that cmake can generate. Unless .vs is produced when cmake is run, it shouldn't be there